### ISSUE TITLE
Create a point-of-sale

### ISSUE THREAD

**Issue by nshoap-utk** [QT-Pies/inventory-project#30]
# Point of Sale
As a developer, it's important that our program is tested with real world conditions.  In addition to the generators we have, it would be nice to have an actual point of sale experience to test the manager with.

## Acceptance Criteria
1. Point of Sale can export information to inventory manager and update inventory.

## Implementation Information
1. Create a class that acts as a point of sale, name it something like ```PointOfSale```.
2. The InventoryManager class should have a function that creates a ```PointOfSale``` object.
3. We can use a command line argument to tell the InventoryManager we want PointOfSale mode, or we can have it read from standard input like it does for other commands.  Or both.
4. The ```PointOfSale``` class should read input from standard input, and it should support commands like:
- ```ADD_TO_SALE item_id``` should check that an item with ID exists, and add it to the SaleList if so.
- ```PURCHASE``` should end the transaction.
5. Once the transaction is ended, InventoryManager should process the SaleList.
6. After this, InventoryManager should continue in PointOfMode until it receives a ```EXIT``` command to return to normal InventoryManager mode.

**Comment by nshoap-utk** [comment]
Moving to sprint 4

**Comment by nshoap-utk** [comment]
I'll also claim this one for myself since it's unassigned

**Comment by nshoap-utk** [comment]
I'll assign this to myself for now, but since I'm gonna be doing GUI work feel free to reassign.

**Comment by nshoap-utk** [comment]
Jen and I previously talked about me writing up a header file for what we want the POS to look like, and letting her implement it.  So we'll go with that plan.

**Comment by jhulen** [comment]
As discussed in the server, we decided making a POS was making things unnecessarily difficult. I'll paste Noah's idea that I implemented here.

"I was looking over https://github.com/QT-Pies/inventory-project/issues/30 this to start creating the header for Jen to use, and honestly, I feel like we don't really need this.  It's going to essentially be moving the sales part of the IM over to another class.

She can just change the existing code to stay in sale mode until we specify we're done making transactions. What I am suggesting is we just make it so the program is like:
Enter a transaction
End this transaction
and then loops back to enter transaction until we exit from this screen.

It seems like we're gonna be moving stuff for no real reason (outside of needing commits) if we do this, so I'd consider just modify what we already have a bit, and try to think of another issue to work on instead."
:)
